keyboards/qneatandtidy/dx65/hotswap/rev1/info.json

```diff
@@ -0,0 +1,211 @@
+    },
+    "ws2812": {
+        "driver": "vendor",
+        "pin": "GP12"
+    },
+    "community_layouts": ["65_ansi_blocker"],
+    "layouts": {
+        "LAYOUT_65_ansi_blocker": {
+            "layout": [
+                {"label": "Esc", "matrix": [0, 0], "x": 0, "y": 0},
+                {"label": "1", "matrix": [0, 1], "x": 1, "y": 0},
+                {"label": "2", "matrix": [0, 2], "x": 2, "y": 0},
+                {"label": "3", "matrix": [0, 3], "x": 3, "y": 0},
+                {"label": "4", "matrix": [0, 4], "x": 4, "y": 0},
+                {"label": "5", "matrix": [0, 5], "x": 5, "y": 0},
+                {"label": "6", "matrix": [0, 6], "x": 6, "y": 0},
+                {"label": "7", "matrix": [0, 7], "x": 7, "y": 0},
+                {"label": "8", "matrix": [0, 8], "x": 8, "y": 0},
+                {"label": "9", "matrix": [0, 9], "x": 9, "y": 0},
+                {"label": "0", "matrix": [0, 10], "x": 10, "y": 0},
+                {"label": "Minus", "matrix": [0, 11], "x": 11, "y": 0},
+                {"label": "Equal", "matrix": [0, 12], "x": 12, "y": 0},
+                {"label": "Backspace", "matrix": [0, 13], "x": 13, "y": 0, "w": 2},
+                {"label": "Home", "matrix": [0, 14], "x": 15, "y": 0},
+                {"label": "Tab", "matrix": [1, 0], "x": 0, "y": 1, "w": 1.5},
+                {"label": "Q", "matrix": [1, 1], "x": 1.5, "y": 1},
+                {"label": "W", "matrix": [1, 2], "x": 2.5, "y": 1},
+                {"label": "E", "matrix": [1, 3], "x": 3.5, "y": 1},
+                {"label": "R", "matrix": [1, 4], "x": 4.5, "y": 1},
+                {"label": "T", "matrix": [1, 5], "x": 5.5, "y": 1},
+                {"label": "Y", "matrix": [1, 6], "x": 6.5, "y": 1},
+                {"label": "U", "matrix": [1, 7], "x": 7.5, "y": 1},
+                {"label": "I", "matrix": [1, 8], "x": 8.5, "y": 1},
+                {"label": "O", "matrix": [1, 9], "x": 9.5, "y": 1},
+                {"label": "P", "matrix": [1, 10], "x": 10.5, "y": 1},
+                {"label": "Left Bracket", "matrix": [1, 11], "x": 11.5, "y": 1},
+                {"label": "Right Bracket", "matrix": [1, 12], "x": 12.5, "y": 1},
+                {"label": "Slash", "matrix": [1, 13], "x": 13.5, "y": 1, "w": 1.5},
+                {"label": "PgUp", "matrix": [1, 14], "x": 15, "y": 1},
+                {"label": "Caps Lock", "matrix": [2, 0], "x": 0, "y": 2, "w": 1.75},
+                {"label": "A", "matrix": [2, 1], "x": 1.75, "y": 2},
+                {"label": "S", "matrix": [2, 2], "x": 2.75, "y": 2},
+                {"label": "D", "matrix": [2, 3], "x": 3.75, "y": 2},
+                {"label": "F", "matrix": [2, 4], "x": 4.75, "y": 2},
+                {"label": "G", "matrix": [2, 5], "x": 5.75, "y": 2},
+                {"label": "H", "matrix": [2, 6], "x": 6.75, "y": 2},
+                {"label": "J", "matrix": [2, 7], "x": 7.75, "y": 2},
+                {"label": "K", "matrix": [2, 8], "x": 8.75, "y": 2},
```

Comment:
this layout is missing (from top to bottom, left to right):
1. latter 4 keys on 3rd row (`"y":2`) 
2. entire 4th row (`"y:3"`)
3. first key on 5th row (`"y:4"`)